Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -5,9 +5,9 @@ title: Prerequisites
 Rook can be installed on any existing Kubernetes cluster as long as it meets the minimum version
 and Rook is granted the required privileges (see below for more information).
 
-## Minimum Version
+## Kubernetes Version
 
-Kubernetes **v1.22** or higher is supported.
+Kubernetes versions **v1.22** through **v1.29** are supported.
```

Comment:
Actually the min version supported is 1.23 now. We have this listed as a breaking change in the [upgrade guide](https://rook.io/docs/rook/latest/Upgrade/rook-upgrade/#breaking-changes-in-v113), but missed updating these other docs. So let's change it to 1.23, thanks.

Reply:
Updated